src/python_testing/TC_pics_checker.py

```diff
@@ -159,7 +159,14 @@ def test_TC_IDM_10_4(self):
                 # Codegen in python uses feature masks (0x01, 0x02, 0x04 etc.)
                 # PICS uses the mask bit number (1, 2, 3)
                 # Convert the mask to a bit number so we can check the PICS.
-                feature_bit = int(math.log2(feature_mask))
+                try:
+                    feature_bit = int(math.log2(feature_mask))
+                except ValueError:
+                    location = FeaturePathLocation(endpoint_id=self.endpoint_id,
+                                                   cluster_id=cluster_id, feature_code=str(feature_mask))
+                    self.record_warning("PICS check", location=location,
```

Comment:
Should probably have a comment explaining why this is a warning and not a fatal error?

Reply:
added.